server/dataset/cxg_dataset.py

```diff
@@ -393,7 +427,13 @@ def _get_schema(self):
         annotations = {}
         for ax in ("obs", "var"):
             A = self.open_array(ax)
-            schema_hints = json.loads(A.meta["cxg_schema"]) if "cxg_schema" in A.meta else {}
+
+            if ax == "obs":
+                # Filter out int64 values in the obs cxg_schema metadata
+                schema_hints = self._get_filtered_cxg_schema_metadata()
```

Comment:
not sure if it's a problem to open up the `obs` array twice, both in `A = self.open_array(ax)` and again in `self._get_filtered_cxg_schema_metadata()`

i would expect tiledb can handle concurrent reads without blowing up, but it could result in performance being less efficient.